CRM/Civicase/Hook/BuildForm/TokenTree.php

```diff
@@ -79,9 +86,38 @@ private function attachNewTokenTreeToForm(CRM_Core_Form &$form) {
       }
     }
     $this->reFormatCustomTokens($newTokenTree);
+    $newTokenTree = $this->reOrderTokens($newTokenTree);
     $this->addTokensToJs($newTokenTree);
   }
 
+  /**
+   * Reorder the tokens.
```

Comment:
Can you add a bit more description to the function so it's a bit obvious what is doing.

Reply:
changed